logs/scraper_run.md

```
@@ -0,0 +1,4 @@
+2025-08-31 12:24:43 - INFO - [root] - Iniciando crawling para: ['https://code.visualstudio.com/']
+2025-08-31 12:24:47 - INFO - [src.llm_extractor] - LLMExtractor inicializado con el cliente de OpenAI parcheado.
+2025-08-31 12:25:04 - WARNING - [ScrapingOrchestrator] - Se ha desactivado la comprobación de robots.txt.
+2025-08-31 12:25:04 - INFO - [ScrapingOrchestrator] - Trabajador 1 procesando: https://code.visualstudio.com/ (Prioridad: -1)
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)